src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -546,16 +546,19 @@ doclet.usage.tag.description=\
     Specify single argument custom tags
 
 doclet.usage.taglet.description=\
-    The fully qualified name of Taglet to register
+    The fully qualified name of taglet to register
```

Comment:
`taglet` should have an article, so, suggest: `The fully qualified name of the taglet to register`